src/nfts/NftMedia.tsx

```diff
@@ -17,7 +17,7 @@ import networkConfig from 'src/web3/networkConfig'
 const DEFAULT_HEIGHT = 360
 
 interface PlaceHolderProps {
-  testID: string
+  testID?: string
```

Comment:
why not just assign a testId for the celebration nft image, rather than make the prop optional? i see some assumptions in the component that relies on this (.e.g `testID={`${testID}/VideoPlaceholder`}`)